src/data_morph/data/stats.py

```diff
@@ -24,12 +25,14 @@ def get_values(df: pd.DataFrame) -> SummaryStatistics:
     Returns
     -------
     SummaryStatistics
-        Named tuple consisting of mean and standard deviations of x and y,
+        Named tuple consisting of mean, median and standard deviations of x and y,
```

Comment:
I much prefer Oxford commas, so let's change this to this:

```suggestion
        Named tuple consisting of mean, median, and standard deviations of x and y,
```